packages/swapper/src/swappers/PortalsSwapper/getPortalsTradeQuote/getPortalsTradeQuote.ts

```diff
@@ -114,7 +113,7 @@ export async function getPortalsTradeQuote(
       inputToken,
       outputToken,
       inputAmount: sellAmountIncludingProtocolFeesCryptoBaseUnit,
-      slippageTolerancePercentage: Number(slippageTolerancePercentageDecimal) * 100,
+      slippageTolerancePercentage: maybeSlippageTolerancePercentageOverride,
```

Comment:
leverage user/default terminology in `fetchPortalsTradeOrder` too

Reply:
https://github.com/shapeshift/web/pull/7654/commits/d65f3d5877f925e154a1aec358681b08b2bd04a8